astropy/coordinates/representation/geodetic.py

```diff
@@ -94,10 +139,114 @@ def from_cartesian(cls, cart):
         Converts 3D rectangular cartesian coordinates (assumed geocentric) to
         geodetic coordinates.
         """
+        # Compute geodetic/planetodetic angles
         lon, lat, height = erfa.gc2gde(
             cls._equatorial_radius, cls._flattening, cart.get_xyz(xyz_axis=-1)
         )
-        return cls(lon, lat, height, copy=False)
+        return cls(
```

Comment:
I don't think we should make this sense. We don't have to enforce units of degrees; if people want that, they can convert themselves.